{
	"id": "CVE-2022-36283",
	"sourceIdentifier": "secure@intel.com",
	"vulnStatus": "Rejected",
	"published": "2025-05-28T19:15:29.270",
	"lastModified": "2025-05-28T19:15:29.270",
	"descriptions": [
		{
			"lang": "en",
			"value": "Rejected reason: This CVE ID has been rejected or withdrawn by its CVE Numbering Authority because it is Unused"
		}
	],
	"references": [],
	"metrics": {}
}
